src/resources/fonts.c

```diff
@@ -177,6 +177,25 @@ bool fonts_init(void) {
     return false;
 }
 
+void fonts_set_font(font *new_font, font_size font_sz) {
+    switch(font_sz) {
+        case FONT_BIG:
+            memcpy(&font_large, new_font, sizeof(font));
```

Comment:
```suggestion
            font_large = *new_font;
```
(& same for other cases)

Reply:
Done.